### DISCUSSION TITLE
How to disable the clear-button?

### DISCUSSION THREAD

**Discussion by MichaelRinger** [joshhanley/livewire-autocomplete#16] [Help]
Havent found an option to disable the clear button.

**Comment by joshhanley** [comment]
@MichaelRinger hmm I don't think there is an option to disable it just yet.

My reasoning is, if a user has selected a record by mistake then they need a way to clear the selected record, as such would generally need the button do be there.

Could you describe why you would want to disable the clear button? If so then how would you give the user an option to clear/reset the autocomplete?

Thanks!

**Reply by MichaelRinger** [reply]
I think clearing the input via the keyboard delete button is much more natural. Also when something is selected I am forced to click the button cause the input is disabled which is also kinda annoying.

**Reply by joshhanley** [reply]
@MichaelRinger you should be able to use the keyboard delete when entering new text. The input is only disabled when an existing record is selected (from the dropdown).

If the input wasn't disabled, how does the user know that a record has been selected via the dropdown vs a new record is being entered/created?

**Reply by MichaelRinger** [reply]
I am using a mac and chrome as the browser and as soon there is an item in the dropdown when I search, the input gets disabled.
Sry for the late reply.
